bin/name_upgrade.dart

```diff
@@ -36,15 +36,10 @@ Future main(List<String> args) async {
   // The AnalyzerContext we use only contains the content in lib.
   // Move all the code we care about into lib temporarily to upgrade any
   // modules that might live outside of there normally.
-  utils.moveTargetsIntoLib(targets);
-
-  final sdkDir = utils.getSdkDir();
-
-  final context = utils.createAnalysisContext(sdkDir);
+  moveTargetsIntoLib(targets);
 
-  utils
-      .groupClassesBySource(utils.moduleClassesWithoutNames(context, sdkDir))
-      .forEach(utils.writeGettersForSource);
+  final classes = getModulesWithoutNamesBySource();
+  classes.forEach(writeGettersForSource);
 
-  utils.moveTargetsOutOfLib(targets);
+  moveTargetsOutOfLib(targets);
```

Comment:
#nit Just thought... Could we put this in a `finally` block so just in case something goes wrong, it doesn't leave the repo in a bad state?

Reply:
I moved it to a finally but it didn't fix the bad state 🤷